<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NightBox</class>
 <widget class="QDialog" name="NightBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NightBox</string>
  </property>
  <widget class="NLabelButton" name="aboutLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>499</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;关于&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="systemPushButton">
      <property name="text">
       <string>系统操作</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mathPushButton">
      <property name="text">
       <string>数学工具</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exPushButton">
      <property name="text">
       <string>答案穷举</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NLabelButton</class>
   <extends>QLabel</extends>
   <header>nlabelbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
